quickstarts/quickstart-template-guide.md

```diff
@@ -0,0 +1,141 @@
+
+Use this section to provide:
+- Basic instructions/commands to install your application.
+ - Always validate the commands and check for technical accuracy with your engineering team.
+ - Provide instructions to verify that the installation is successful.  
+- Link to the detailed installation guide if applicable.
+- Links to the upstream docs for common software installation instructions.
+
+## About the "Steps" section
+
+The steps section is where you describe what the user needs to do. How you write your steps will vary depending on your organization's style guide.
+This template breaks down the quickstart into **parts**. One part of the quickstart may focus on completing several related steps. You're welcome to follow this structure or use the step headings on their own.
+
+If your quickstart involves many complex tasks then break it down into different logical parts with each part consisting of one or more related steps. For example, structure a quickstart guide to onboard an app as follows:
+Part 1 Configure your local dev environment
+Step 1.1 
+Step 1.2
+Part 2 Build your app
+Step 2.1
+Step 2.2
+Part 3 Deploy your app 
+Step 3.1
+Step 3.2
+Part 4 Test your app 
+Step 4.1
+Step 4.2
+
+However, if your quickstart does not involve many independent tasks, we recommend that you add only the logical steps. For example, structure  a quickstart guide to create a new project in Visual Studio as follows:
+
+Step 1 Create a new project
+Step 2 Select a template type
+Step 3 Configure your new project
+
+### Tips for writing steps in a quickstart
+
+- Number the steps in the format {part}.{step} as it helps your users to easily identify and locate procedures in the quickstart. Also, this helps in referencing a particular step in a lengthy quickstart.
+- Start the heading with a verb and express the step/part headings as a complete thought. For example, you could use the heading **Connect to the VM instance** instead of **Connect**.   
+- For each step, provide some background information about the task so users know what they're about to do and why.
+- Remember to orient your users when walking them through each step. If they need to open a particular file or dialog to complete the task, provide that information first.
+- Use plain language and define the terminology of any technical term next to it.
+- Include one action in a step.
+
+:information_source: For additional tips on writing steps, see [Writing Procedural Steps](writing-tips.md#writing-procedural-steps) from **The Good Docs Project**.
+
+## About the "Next steps" section
+
+- Use this section to provide links to other tutorials/articles that the users can try on completing the quickstart. 
+- Consider a logical connect from the current quickstart that can act as a basis for your users’ next learning.  
+- (Optional) Provide links to relevant resources, like blogs, reference docs, videos, how-tos, and so on under a new heading **See Also**. 
+
```

Comment:
**Suggestion (new section, non blocking):** Have you thought about providing a mini-troubleshooting section in a quickstart? For example, I was recently working through a Python quickstart tutorial and I liked how they gave a troubleshooting guide at the end: https://realpython.com/interacting-with-python/

Check out the part that says, "Did something wrong? Perhaps you made one of these mistakes..."

Reply:
We have researched a bit and discussed on this topic. Majority of quickstarts do not have troubleshooting section ( [GitHub](https://docs.github.com/en/get-started/quickstart/hello-world), [gcloud](https://cloud.google.com/sdk/docs/quickstart), [Salesforce](https://developer.salesforce.com/docs/platform/functions/guide/quick-start.html)) and they do have a page as an overall documentation. We felt it would be more appropriate for a README doc where it introduces and explains a project